docs/plugins.md

```diff
@@ -85,64 +85,185 @@ the output of Protractor by returning a results object.
 Plugins are node modules which export an object with the following API:
```

Comment:
@andresdominguez Could you look through this and see if these plugin expansions will be sufficient for your needs?